### DISCUSSION TITLE
How to create an indexedDB composite key

### DISCUSSION THREAD

**Discussion by AdiTasa** [cyrilletuzi/angular-async-local-storage#1006] [Q&A] [answered]
Hey
How do I create a composite key in indexeddb?

something like this `var store = db.createObjectStore('myStore', 
    {keyPath: ['id1', 'id2']}
);`

thanks

**Comment by cyrilletuzi** [comment, marked as answer]
Hello,

That is not supported by this library, which aims to provide a simple API and be widely compatible (this case would be difficult to support in `localStorage` fallback for example).

Advanced cases should use `indexedDb` directly, or use other libraries.
